clustering/server/src/main/resources/org.wildfly.clustering.server.provider.proto

```diff
@@ -5,12 +5,6 @@ import "org.jgroups.util.proto";
 
 // IDs: 140 -144
 
-/**
- * @TypeId(140)
- */
-message GetLocalServicesCommand {
-}
```

Comment:
I am under impression that proto types should be commented out rather than removed so the IDs don't get inadvertently reused?

Reply:
We are not yet guaranteeing serialization compatibility across major releases.